third_party/rust/patches/serialport-pci-workaround.patch

```diff
@@ -1,8 +1,8 @@
 diff --git a/src/posix/enumerate.rs b/src/posix/enumerate.rs
-index 65c42bb..ee50f55 100644
+index bec8f86...0000000 100644
```

Comment:
FYI @lschuermann, now I remember you mentioned that this patch didn't apply cleanly. Thanks @nbdd0121 for updating it!